packages/expo-google-sign-in/src/index.js

```diff
@@ -0,0 +1,337 @@
+// @flow
```

Comment:
Would you care to convert this module to TypeScript? 🙏

Reply:
we're just about to add this with https://github.com/expo/expo/pull/2643 . Let's see if that lands soon but I also don't want this PR to be blocked on it